props/saber_sabersense_buttons.h

```diff
@@ -720,7 +735,7 @@ class SabersenseButtons : public PROP_INHERIT_PREFIX PropBase {
   }
 #endif
 
-  void NextBladeArray() {
+  void IdentBladeArray() {  // Plays associated arrayx.wav to ident array landed on.
```

Comment:
If this is what it does, how about calling it PlayArraySound() ?
Seems like if we name it well, no comment is actually required.